policy/feature-flags.md

```diff
@@ -6,63 +6,137 @@ Product and release need to track bugs whose visibility is controlled through a
+  <dt>releaseNN</dt>
+  <dd>The feature is enabled by default in Firefox NN (Release), Beta and Nightly</dd>
+  <dt>betaNN+1</dt>
+  <dd>The feature is enabled by default in FirefoxNN+1 (Beta), and Nightly, but not Release</dd>
+  <dt>nightlyNN+2</dt>
+  <dd>The feature is enabled by default in FirefoxNN+2 (Nightly), but neither Beta or Release</dd>
+</dl>
+
+#### Maintenance 
+
+If the current values of the flag were:
+
+- `release60`
+- `beta61`
+- `beta62`
+
+on merge day we would add
+
+- `release61`
+- `beta62`
+- `nightly63`
+
+and disable (but not delete) `release60`, `beta61`, and `nightly62`.
+
+#### ESR
+
+For tracking the feature in ESR, we create a `behind-pref-esr` status flag. It will be kept up with the values of the current and previous ESR releases. 
+
+_Example_
+
+- `---`
+- `off`
+- `esr59`
+- `esr60`
 
 ### Example
 
-A bug is filed, "Make Tabby Cats the default new tab experience." 
+A bug is filed, "Make Tabby Cats the default new tab experience." And the team developing this (engineering and product) decide that this should be controlled behind a preference, `browser.newtabpage.default.tabbycat`. The developers break the work for this feature down into three bugs.
+
+- The bug's summary is updated to `[meta] Make Tabby Cats the default new tab experience`
+- A comment is filed listing the name of the preference
+- The `behind-pref` flag is set to `in-progress`
+- The bug's severity is set to `enhancement`
+- The three implementation bugs should be marked as blocking the `[meta]` bug for the new feature
+
+As the feature is developed and the individual patches implement it land, it's kept off by compiler directives, the pref, or both. As these land, and are not backed out, these bugs can be marked RESOLVED FIXED.
+
+Once the dependent bugs have been resolved, the feature can be enabled in nightly. The lead for the feature–which may be an engineer, a program manager, or a product manager–must notify the Nightly Release Manager before enabling it.
+
+- The `[meta]` bug's status is moved to RESOLVED and resolution to FIXED.
+- The bug's `behind-pref` flag is set to `nightlyNN+2` to indicate it's now available in nightly
+- The `qe-verify` flag is set to ?, requesting QA's attention
+
+Before the feature can graduate to Beta, it must be verified by QA. 
+
+- QA agrees to take on verfication, setting `qe-verify` to `+`
+- The feature is tested on nightly and confirmed to work as specified (implicit here is the feature team's involvement in creating a test plan)
+- QA moves the bug's resolution to VERIFIED
+
+If the feature does not pass testing then QA should file bugs blocking the `[meta]` bug for the feature. QA and the development team must confer and decide if the feature will be disabled in Nightly, or allowed to be kept on while bugs are fixed. This will depend on risk and severity of the bugs found. 
 
-It's decided that this should be controlled behind a preference, `browser.newtabpage.default.tabbycat`:
```

Comment:
It's not clear in which bug the commits should be reviewed and landed to enable/disable the prefs. Normally there should be one landing per bug e.g. only enabling… disabling in Nightly would be a separate bug so that the target milestone reflects when it merges. Would we still use one landing per bug which would mean landing in the non-meta bug?